gst.so/src/gst_worker.cpp

```diff
@@ -150,7 +151,7 @@ void GSTWorker::hit_max_gflops(int *error, string *err_description) {
 
     // Waits for GEMM operation to complete
     if(!gpu_blas->is_gemm_op_complete())
-      continue;
+      continue;  // failed to run the GEMM operation
```

Comment:
dont we need to break out?

Reply:
This function is obsolete. Hence, didn't change it.